### PR TITLE
rdw005

### PR DESCRIPTION
**Pull request by thea-m**
I would be grateful if you could have a look at the encoding of this scroll, in which I have tried to incorporate the outcomes of the many scroll issues of the past.

I have two specific questions:
- the locus in ms1 is commented out, because it leads to an error message as it doesn't start with a digit. Does the schema need to be updated or the guidelines?
- I have (out of habit) numbered the lines continuously through the entire scroll, not section by section or text by text. What would be preferable here?

Thank you!

### CODE REVIEW

File: Hamburg/HLC/RDW005.xml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                              <height>450</height>
+                              <width>45</width>
+                           </dimensions>
+                        </extent>
+                        <!--<condition key=""></condition>-->
+                     </supportDesc>
+                     
+                        <layoutDesc>
+                           <layout columns="1" writtenLines="">
+                              <note>Number of characters per line: </note>
```
Comment (DenisNosnitsin1970) [inline Hamburg/HLC/RDW005.xml]: No numbers, here and below?

File: Hamburg/HLC/RDW005.xml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                                 <dimensions unit="mm" xml:id="textarea1">
+                              <height>425</height>
+                              <width>39</width>
+                           </dimensions>
+                           </layout>
+                           
+                           
+                           <layout columns="1" writtenLines="">
+                              <note>Number of characters per line: </note>
```
Comment (DenisNosnitsin1970) [inline Hamburg/HLC/RDW005.xml]: Where text area 1 end? Where text area 2 starts?

File: Hamburg/HLC/RDW005.xml [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                        <dimensions>
+                           <height>75</height>
+                           <width>46</width>
+                        </dimensions>
+                     </decoNote>
+                  </decoDesc>
+
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <decoNote xml:id="b1" type="SlipCase">The scroll is kept in a case probably made of reptile skin.</decoNote>
```
Comment (DenisNosnitsin1970) [inline Hamburg/HLC/RDW005.xml]: Perhaps a more specific term can be used to refer to the traditional container (?) used for keeping Ethiopic scrolls. Is it European- or Ethiopian-made?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
@DenisNosnitsin1970 , thank you! But you accidentally looked at an older version of the file, could you check the latest one (this has the characteres per line and text area correspondences):
 https://github.com/BetaMasaheft/Manuscripts/pull/1182/files
I think that the container is Ethiopian-made. Do you know a more specific term?
image
image

File: Hamburg/HLC/RDW005.xml [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+                  <div type="textpart" subtype="text" n="3" corresp="#ms_i3" xml:id="text3">
+                  <ab>
+                     <lb n="64"/>ጸሎተ፡ ነደ
+                     <lb n="65"/>ራ፡ ወእንዘ
+                     <lb n="66"/>የሐውር፡ እ
+                     <lb n="67"/>ግዚእነ፡ ኢ
+                     <lb n="68"/>የሱስ፡ ክር
+                     <lb n="69"/>ስቶስ፡ ውስ
+                     <lb n="70"/>ተ ባህረ፡ ጥ
+                     <lb n="71"/>ብርያይስ፡ ወ
```
Comment (DenisNosnitsin1970) [inline Hamburg/HLC/RDW005.xml:293]: probably it is ጥብርያዶስ or ደስ

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
Strelcyn uses "cylindrical leather case", or "(beautiful) cylindrical silver case", or "cylindrica case made of .... (covered with)". I wonder what is a traditional Amharic or TGN term for this (Amhars would possibly say መያዣ, but it is "container", there may be a specific word for this, we have to ask around). 
It is indeed the first case that I see a part of a manuscript made of such a skin looking like a reptile skin.  
l. 52: ቁመኛ or ቂመኛ?

**Comment by thea-m** [comment]
Thank you, I checked the manuscript and carried out a few corrections in the text.
@DariaElagina and @PietroLiuzzo , it would be great if you could also have a look :)

**Comment by PietroLiuzzo** [comment]
from the guidelines 
- https://betamasaheft.eu/Guidelines/?q=referencing&id=referencing-the-manuscript 
- https://betamasaheft.eu/Guidelines/?q=scroll&id=scrolls

I think you can choose from several options for referencing
- using `<locus>`, in which case, you do need to state a `<pb n='1r'>`
````xml
 <msItem xml:id="ms_i1">
                     <locus from="1r1" to="1r41"/>
                     <title type="complete" ref="LIT6473PPAynaTela"/>
                    <textLang mainLang="gez"/>
                     <note>See the transcription of the incipit and explicit in <ref target="#text1"/>.</note>
                  </msItem>  
````

- using `<ref type="mss" corresp="RDW005.recto.sec2.1"/>` which however is not allowed in `<msItem>`.
````xml
 <msItem xml:id="ms_i1">
                     <title type="complete" ref="LIT6473PPAynaTela"/>
                    <textLang mainLang="gez"/> 
                     <note>
<ref type="mss" corresp="RDW005.recto.sec2.1-41"/>
See the transcription of the incipit and explicit in <ref target="#text1"/>.
</note>
                  </msItem>  
````

**Comment by PietroLiuzzo** [comment]
The numbering of lines is IMHO not really an issue, if it has been done one way, it can stay that way without affecting the encoding or its reuse

**Comment by thea-m** [comment]
Thank you, Pietro!

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**